core/jvm/src/main/scala/me/shadaj/scalapy/interpreter/Platform.scala

```diff
@@ -79,4 +81,9 @@ object Platform {
   def setPtrLong(ptr: Pointer, offset: Int, value: Long): Unit = ptr.setLong(offset, value)
   def setPtrInt(ptr: Pointer, offset: Int, value: Int): Unit = ptr.setInt(offset, value)
   def setPtrByte(ptr: Pointer, offset: Int, value: Byte): Unit = ptr.setByte(offset, value)
+
+  def programName: Option[jna.WString] =
+    Properties.propOrNone("scalapy.python.programname")
+      .orElse(sys.env.get("SCALAPY_PYTHON_PROGRAMNAME"))
```

Comment:
Shouldn't we do the opposite, that is have the env var override the property? In my experience, properties tend to be used programmatically (by scripts or from build tools, for example). And env vars allow users to override those values, if they want to.

That's how it works in [coursier](https://get-coursier.io/docs/cache#manual-override) for example, and everyone seems happy with it.